index.html

```diff
@@ -10,228 +10,307 @@
+                        Glad to see<br>you!</h1>
+                    <p class="description">I’m a software developer! I can help you build a product , feature or website
+                        Look through some of my work and experience! If you like what you see and have a project you
+                        need coded, don’t hestiate to contact me.</p>
+                </div>
+
+                <div class="social container">
+                    <a href="#" class="button connect">LET’S CONNECT</a>
+                    <ul class="social-icons">
+                        <li><img src="/images/social/Twitter.svg" alt=""></li>
+                        <li><img src="/images/social/Linkedin.svg" alt=""></li>
+                        <li><img src="/images/social/Vector-M.svg" alt=""></li>
+                        <li><img src="/images/social/Github.svg" alt=""></li>
+                        <li><img src="/images/social/Hand2.svg" alt=""></li>
                     </ul>
+                </div>
+            </div>
+        </section>
+
+        <section class="works card-1">
+            <div class="card-container-1">
+                <img class="img-version-mobile" src="/images/Snapshoot1.png" alt="Snapshot Portfolio Image">
+                <img class="img-version-desktop" src="/images/img-desktop-1.png" alt="Snapshot Portfolio Image">
+            </div>
 
-                    <button type="button" class="see-proj1">See Project</button> 
+            <div class="grid-container-desktop">
+                <div class="title experience">
+                    <h1 class="mobile-title">Tonic</h1>
+                    <h1 class="desktop-title">Tonic</h1>
+                    <ul class="flex-container-list">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
                 </div>
-            </li>
-            <!-- Second Card -->
-            <li class="card-container2">
-                <img src="./images/Snapshoot2.png" class="card-container-img2" alt="Snapshoot2">
+                <div class="paragraph">
+                    <p class="mobile-para">A daily selection of privately<br> personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <p class="desktop-para">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                </div>
+                <div class="languages">
+                    <ul class="flex-lang">
+                        <li><span>html</span></li>
+                        <li><span>css</span></li>
+                        <li><span>javaScript</span></li>
+                    </ul>
+                </div>
+                <div class="btn-see-projects">
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+        </section>
```

Comment:
Since you didn't followed the recommendations given for the mobile design, so you will be having problems in aligning this section.
![image](https://user-images.githubusercontent.com/61361037/221282124-6608a940-822e-43fc-97bd-316031df9c05.png)

1. The works section was supposed to be in 1 `<section>` tag only.
2. That would be using grid to align all the work cards

So, I will again recommend you to understand the logic:
A. Grid would act like container with specific width and height rules, that change as the size of the page changes, keeping the dimensions of the grid items.
B. Put the work cards in one `section` tag and apply grid to align it both for mobile and desktop design. This is essential for responsiveness of this section.